ProcessManager.php

```diff
@@ -723,7 +723,7 @@ protected function bootstrapFailed(Connection $conn)
 
             foreach ($this->slaves as &$slave) {
                 $slave['keepClosed'] = true;
-                if(true === isset($slave['connection']) && true === is_object($slave['connection'])) {
+                if(true == isset($slave['connection']) && true == is_object($slave['connection'])) {
```

Comment:
I guess `if(isset($slave['connection']) && null !== $slave['connection'])` 
can be reduced to `if (!empty($slave['connection']))`.